src/test/suite/insert-mode-additions.test.ts

```diff
@@ -13,7 +13,7 @@ import {
     openTextDocument,
     sendInsertKey,
     sendVSCodeCommand,
-} from "../utils";
+} from "../integrationUtils";
```

Comment:
looks like unit tests live in `src/test/unit/` , which is good. so probably integ tests should live in `src/test/integ/`.

and if this util module is only for integ tests, should we move it into `src/test/integ/` ?

Reply:
Yeah sure. I waffled on this, but I'll get that (and associated docs) cleaned up